docs/modules/dome.md

```diff
@@ -80,6 +80,9 @@ Returns the generic name of the operating system or platform, where possible.
 #### `time: Number`
 Returns the integer number of seconds since the unix epoch (1st January 1970).
 
+#### `screenCount: Number`
```

Comment:
This should be displayCount, and "number of displays"

Reply:
It should. Whoops

Reply:
And now its like it never happened ;P